efel/cppcore/LibV5.cpp

```diff
@@ -3660,3 +3782,844 @@ int LibV5::time_to_interburst_min(mapStr2intVec& IntFeatureData,
+  if (retVal <= 0) return -1;
+
+  // Get time
+  retVal = getVec(DoubleFeatureData, StringData, "T", t);
+  if (retVal <= 0) return -1;
+
+  retVal = getVec(IntFeatureData, StringData, "peak_indices",
+                  peak_indices);
+  if (retVal < 0) return -1;
+
+  retVal = getVec(IntFeatureData, StringData, "burst_end_indices",
+                  burst_end_indices);
+  if (retVal < 0) return -1;
+
+  // Get postburst fash ahp indices
+  retVal = getVec(IntFeatureData, StringData, "postburst_fast_ahp_indices", postburst_fast_ahp_indices);
+  if (retVal < 1) {
+    GErrorStr +=
+        "\n At least one post-burst fast AHP required for calculation of "
+        "interburst_25percent_indices.\n";
+    return -1;
+  }
+
+  retVal =
+      __interburst_25percent_indices(t, v, postburst_fast_ahp_indices, peak_indices, burst_end_indices,
+                                   interburst_25percent_indices, interburst_25percent_values);
+
+  if (retVal == 0)
+    return -1;
+  if (retVal > 0) {
+    setVec(IntFeatureData, StringData, "interburst_25percent_indices", interburst_25percent_indices);
+    setVec(DoubleFeatureData, StringData, "interburst_25percent_values",
+                 interburst_25percent_values);
+    return interburst_25percent_indices.size();
+  }
+  return -1;
+}
+
+int LibV5::interburst_25percent_values(mapStr2intVec& IntFeatureData,
+                                 mapStr2doubleVec& DoubleFeatureData,
+                                 mapStr2Str& StringData) {
+  return 1;
+}
+
+// index and voltage value at 30% of the duration of the interburst after fast AHP
+static int __interburst_30percent_indices(const vector<double>& t, const vector<double>& v,
+                                        const vector<int>& postburst_fast_ahp_indices,
+                                        const vector<int>& peak_indices,
+                                        const vector<int>& burst_end_indices,
+                                        vector<int>& interburst_30percent_indices,
+                                        vector<double>& interburst_30percent_values) {
+
+  double time_interval, time_at_30percent;
+  size_t index_at_30percent;
+  for (size_t i = 0; i < postburst_fast_ahp_indices.size(); i++) {
+    if (i < burst_end_indices.size()){
+      if (burst_end_indices[i] + 1 < peak_indices.size()){
+        time_interval = t[peak_indices[burst_end_indices[i] + 1]] - t[postburst_fast_ahp_indices[i]];
+        time_at_30percent = t[postburst_fast_ahp_indices[i]] + time_interval * 0.3;
+        index_at_30percent =
+          distance(t.begin(),
+                    find_if(t.begin(), t.end(),
+                            std::bind2nd(std::greater_equal<double>(), time_at_30percent)));
+        interburst_30percent_indices.push_back(index_at_30percent);
+        interburst_30percent_values.push_back(v[index_at_30percent]);
+      }
+    }
+  }
+  return interburst_30percent_indices.size();
+}
```

Comment:
Actually if this function just takes one more parameter that is the percentage (0.3 here, 0.25 in the other call) then it can be reused in 6 placed.